python-template/{{cookiecutter.placeholder_repo_name}}/setup.py

```diff
@@ -36,17 +36,43 @@ def load_requirements(*requirements_paths):
     with -c in the requirements files.
     Returns a list of requirement strings.
     """
+    by_canonical_name = {}
+
+    def check_name_consistent(package):
+        """
+        Raise exception if package is spelled different ways.
+
+        This ensures that packages are spelled consistently so we can match
```

Comment:
It's not really spelling. 
```suggestion
        This ensures that packages are named consistently so we can match
```